### DISCUSSION TITLE
How to build man docs?

### DISCUSSION THREAD

**Discussion by p1r473** [BurntSushi/ripgrep#2801] [Q&A] [answered]
Hi, the git comes with rg.1.txt.tpl
How do I build it? Cant find it in the install directions
Ive tried a few things but none look perfect:

```
pandoc -s -t man doc/rg.1.txt.tpl -o doc/rg.1
```
```
cp doc/rg.1 /usr/share/man/man1/
```
```
rst2man doc/rg.1
```

```
pandoc rg.1.txt.tpl -s -t man | man -l -
```

**Comment by BurntSushi** [comment, marked as answer]
Yes, see: https://github.com/BurntSushi/ripgrep/blob/master/FAQ.md#does-ripgrep-have-a-man-page

The man page is also in the release artifacts. And since the release artifacts are generated by GitHub Actions, you can look at the workflow files for the precise details. (This is a good general trick for many things.) For example:

https://github.com/BurntSushi/ripgrep/blob/bb8601b2bafb5e68181cbbb84e6ffa4f7a72bf16/.github/workflows/release.yml#L339

The `rg.1.txt.tpl` file you found is unused as of ripgrep 14. I meant to remove it. It is removed now so hopefully it doesn't cause more confusion.

**Reply by p1r473** [reply]
Awesome!
